modules/aleph/src/Aleph/Handler/AlephPatronHandler.php

```diff
@@ -146,10 +147,12 @@ public function getReservations() {
         $request = new AlephRequest();
         $material = new AlephMaterial();
 
+        $end_request_date = (string) $hold_request->xpath('z37/z37-end-request-date')[0];
+
         $request->setStatus((string) $hold_request->xpath('z37/z37-status')[0]);
         $request->setPickupLocation((string) $hold_request->xpath('z37/z37-pickup-location')[0]);
         $request->setOpenDate((string) $hold_request->xpath('z37/z37-open-date')[0]);
-        $request->setEndRequestDate((string) $hold_request->xpath('z37/z37-end-request-date')[0]);
+        $request->setEndRequestDate(DateTime::createFromFormat('Ymd', $end_request_date));
```

Comment:
Should this not be `ALEPH_DATE`?

Reply:
Nope, this is the format we create the object from.
It is provided as `Ymd` and should be converted to `Y-m-d`.

Reply:
So we send dates to Aleph in one format `Y-m-d` and recieve them in another `Ymd`?